classes/05class/exercises/c05-3m01/mrcsmonteiro/Makefile

```diff
@@ -0,0 +1,24 @@
+COMPOSE_RUN = docker-compose run --rm s3
+BUCKET_NAME ?=
+
+create_bucket: 
+	@read -p "Bucket name: " BUCKET_NAME; \
+	export BUCKET_NAME; \
```

Comment:
with the idea of avoiding much scripting in the Makefile, you can export the BUCKET_NAME once in line 2